### PR TITLE
docker-diff: add page

### PR DESCRIPTION
**Pull request by aniketpuro**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/docker-diff.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# docker diff
+
+> Inspect changes to files or directories on a container's filesystem.
+> More information: <https://docs.docker.com/engine/reference/commandline/diff>.
+
+- Show help:
+
+`docker diff`
+
+- Inspect the changes to an container:
```
Comment (gutjuri) [inline pages/common/docker-diff.md:10]:
```suggestion
- Inspect the changes to an container since it has been created:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have one suggestion.

File: pages/common/docker-diff.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# docker diff
+
+> Inspect changes to files or directories on a container's filesystem.
+> More information: <https://docs.docker.com/engine/reference/commandline/diff>.
+
+- Show help:
+
+`docker diff`
```
Comment (kbdharun) [inline pages/common/docker-diff.md:8]:
```suggestion
```

File: pages/common/docker-diff.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Inspect the changes to an container:
+
+`docker diff {{container}}`
```
Comment (kbdharun) [inline pages/common/docker-diff.md:12]:
```suggestion
`docker diff {{container}}`


- Display help:

`docker diff --help`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a suggestion for this page.

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**